packages/snaps-controllers/src/snaps/SnapController.ts

```diff
@@ -979,10 +985,10 @@ export class SnapController extends BaseController<
     await Promise.all(
       Object.entries(blockedSnaps).map(async ([snapId, { status, reason }]) => {
         if (status === SnapsRegistryStatus.Blocked) {
-          return this.#blockSnap(snapId, reason);
+          return this.#blockSnap(snapId as ValidatedSnapId, reason);
```

Comment:
We should probably change the `SnapsRegistry.get` return type instead of casting

Reply:
Yeah, I had done that originally but then changed back to casting because I wasn't sure if I should be touching the types outside of the `SnapController`. I'll make the change!

Reply:
Ah, maybe `Object.entries` doesn't persist the type 🤔